poppy/provider/akamai/background_jobs/check_cert_status_and_update/check_cert_status_and_update_flow.py

```diff
@@ -31,6 +31,7 @@
 
 def check_cert_status_and_update_flow():
     flow = linear_flow.Flow('Update Akamai Property').add(
+        check_cert_status_and_update_tasks.PropertyGetHostnamesTask(),
```

Comment:
why not calling the PropertyGetHostnames class from update_property_tasks

Reply:
Opted for `check_cert_status_and_update_tasks` because we're not updating the property, just checking it. It made more sense to add the class there than to `update_property_tasks`. I think it makes the code easier to follow / more logical.

Reply:
Unless we call the PropertyUpdate class from updatepropertytask.py it should not update the property